js/packages/driver/src/context.ts

```diff
@@ -319,18 +319,16 @@ export class Context<TDriver, TContext, TElement, TSelector> {
   }
 
   async elements(
-    elementOrSelector: TElement | types.Selector<TSelector>,
+    selectorOrElement: TElement | types.Selector<TSelector>,
```

Comment:
Not important at all, I just like it to be consistent :)
```suggestion
    selectorOrElement: types.Selector<TSelector> | TElement,
```